packages/extension-api/src/extension-api.d.ts

```diff
@@ -250,6 +250,8 @@ declare module '@podman-desktop/api' {
   export interface KubernetesProviderConnectionFactory extends ProviderConnectionFactory {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     create?(params: { [key: string]: any }, logger?: Logger, token?: CancellationToken): Promise<void>;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    consilienceCheck?(params: { [key: string]: any }): Promise<string[] | undefined>;
```

Comment:
Hello, I would find another way. I have no clue if I implement something what would be a consilienceCheck

probably it should be an optional parameter of setContainerProviderConnectionFactory method. (or setKubernetesConnectionFactory method)